public/components/common/hooks/use-value-suggestions.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Wazuh app - React hook for getting value suggestions
+ * Copyright (C) 2015-2021 Wazuh, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Find more information about this on the LICENSE file.
+ */
+import { useState, useEffect } from 'react';
+import { getDataPlugin } from '../../../kibana-services';
+import { useIndexPattern } from '.';
+import { IFieldType, IIndexPattern } from 'src/plugins/data/public';
+import React from 'react';
+import {
+  UI_ERROR_SEVERITIES,
+  UIErrorLog,
+  UIErrorSeverity,
+  UILogLevel,
+} from '../../../react-services/error-orchestrator/types';
+import { UI_LOGGER_LEVELS } from '../../../../common/constants';
+import { getErrorOrchestrator } from '../../../react-services/common-services';
+
+export interface IValueSuggestiions {
+  suggestedValues: string[] | boolean[];
+  isLoading: boolean;
+  setQuery: React.Dispatch<React.SetStateAction<string>>;
+}
+
+export const useValueSuggestions = (filterField: string, type: 'string' | 'boolean' = 'string') => {
```

Comment:
Is it possible here you forgot to add the return type of the hook to the interface `IValueSuggestiions`?

Reply:
Indeed I think I did